index.html

```diff
@@ -16,41 +16,47 @@
   <main>
     <body>
       <h1>Quiz-App</h1>
-      <section>
-        <article class="question__sec">
-          <img
-            class="box-img"
-            src="images/bookmark_white_24dp(1).svg"
-            alt="bookmark-icon"
-          />
 
-          <h2>Question</h2>
-          This is where the question is being asked!
+      <article class="question__sec">
+        <img
+          class="box__img"
+          src="images/bookmark_white_24dp(1).svg"
+          alt="bookmark-icon"
+        />
 
-          <button type="button" value="button">Answer</button>
-          <button type="button" value="button">Answer</button>
-
-          <button type="button" value="button">Show Answer</button>
-        </article>
-      </section>
-      <br />
-      <section>
-        <article class="question__sec--part2">
-          <img
-            class="box-img2"
-            src="images/bookmark_white_24dp(1).svg"
-            alt="bookmark-icon"
-          />
-          <h2>Question</h2>
-          This is where the Question is being asked!
-          <form action="Questions">
-            <button type="submit" value="Submit">Answer</button>
-            <button type="submit" value="Submit">Answer</button>
-            <button type="submit" value="Submit">Show-Answer</button>
-          </form>
-        </article>
+        <h2 class="first__header">Question</h2>
+        <p>This is where the question is being asked!</p>
+        <div>
+          <button type="button">Answer</button>
+          <button type="button">Answer</button>
+          <div class="show__btn">
+            <button type="button">Show Answer</button>
+          </div>
+        </div>
+      </article>
+      <section class="tags">
```

Comment:
I think the tags of each question should be part of the parent container. Therefore, I would include them as part of each `article`. :)